bot.py

```diff
@@ -17,37 +18,44 @@
 
 #add readyAt property to bot class
 def readyAtGetter(self):
-	return self._readyAt
+    return self._readyAt
 def readyAtSetter(self,value):
-	self._readyAt = value
+    self._readyAt = value
 commands.Bot.readyAt = property(readyAtGetter, readyAtSetter)
 @bot.event
 async def on_ready():
-	await bot.change_presence(status=discord.Status.online, activity=discord.Game(f'{bot.command_prefix}help for commands'))
-	bot.readyAt = datetime.utcnow()
-	print (f"Bot online")
-	
-'''
-#kind of not a fan of this
+    await bot.change_presence(status=discord.Status.online, activity=discord.Game(f'{bot.command_prefix}help for commands'))
+    bot.readyAt = datetime.utcnow()
+    print (f"Bot online")
+
+
+
 @bot.event
 async def on_command_error(ctx, error):
-	if isinstance(error,commands.errors.CommandNotFound):
-		return
-	await ctx.send("An error occured!\n```{}```".format(error))
-'''
-curdir = os.getcwd()
-curdir.split('\\')
-if "Synergile" not in curdir:
-	os.chdir("Synergile")
-else:
-	pass
+    if isinstance(error,commands.errors.CommandNotFound):
+        return
+    else:
+        channel = bot.get_channel(817127799256252437)
+        with open("error.log", "w") as err:
+            err.write(str(traceback.format_exception(type(error), error, error.__traceback__)))
+        with open("error.log", "rb") as err:
+            file = discord.File(err)
+        log_form = "=====ERROR====\nTime: {0}\nCommand: {1}\nError:\n```{2}```"
+        await channel.send(log_form.format(datetime.utcnow(), ctx.message.content, error), file=file)
+        embed = discord.Embed(title="Something unknown happened!",
+                              description="The issue has been reported",
+                              color=discord.Color.red())
+        await ctx.send(embed=embed)
+        traceback.print_exception(type(error), error, error.__traceback__)
+
+
 for cog in os.listdir(f"{os.getcwd()}{os.path.sep}commands"):#path
-	if cog.endswith(".py"):
-		try:
-			cog = f"commands.{cog.replace('.py', '')}"
-			bot.load_extension(cog)
-		except Exception as e:
-			print(f"{cog} cannot be loaded:")
-			raise e
+    if cog.endswith(".py"):
+        try:
+            cog = f"commands.{cog.replace('.py', '')}"
+            bot.load_extension(cog)
+        except Exception as e:
+            print(f"{cog} cannot be loaded:")
+            raise e
 
 bot.run(token)
```

Comment:
```suggestion
bot.run(token)

```
Add newline, same explanation as above